packages/medusa/src/api/routes/admin/gift-cards/index.ts

```diff
@@ -22,7 +22,7 @@ export default (app) => {
   return app
 }
 
-export const defaultAdminGiftCardFields = [
+export const defaultAdminGiftCardFields: Array<keyof GiftCard> = [
```

Comment:
suggestion(non-blocking): here you are using Array, later you are using [], just for the consistency can we use [] instead?